src/Makefile

```diff
@@ -1,16 +1,22 @@
-# Copyright 2020-2021 Seagate
-#
-# This copyrighted material is made available to anyone wishing to use,
-# modify, copy, or redistribute it subject to the terms and conditions
-# of the GNU General Public License v2 or (at your option) any later version.
-#
+# SPDX-License-Identifier: LGPL-2.1-only
+# Copyright (C) 2021 Seagate Technology LLC and/or its Affiliates.
+
```

Comment:
This patch should be dropped because this patch is to recovery the unexpected changes in the previous patch.

Reply:
Still not sure how to use GitHub. I've created another commit after this commit to change back the Makefile. Or should I be creating another pull request entirely?

Reply:
You don't need to create another new branch, using the command "git push -f" to upload the updated branch onto your github repositorty, the pull request can reflect the updated changes.